python/mxnet/contrib/serde/_export/__init__.py

```diff
@@ -0,0 +1,4 @@
+import onnx
+
+def export_model(sym, params):
+    pass
```

Comment:
?

Reply:
Will delete this module for now.

Reply:
Is this the determined API for exporting MXNet models to onnx format? Have you considered whether we can borrow the interface of this function in Pytorch?
https://github.com/pytorch/pytorch/blob/master/torch/onnx/__init__.py#L43